input/test_cosmo-dd-multispecies.in

```diff
@@ -0,0 +1,105 @@
+	   "HeI_density",
+	   "HeII_density",
+	   "HeIII_density",
+	   "e_density",
+           "metal_density",
+	   "cooling_time",
+	   "temperature",
+	   "gamma"];
+}
+
+Group {
+    list = ["color", "derived"];
+    color {
+       field_list = [
+              "HI_density",
+              "HII_density",
+              "HeI_density",
+              "HeII_density",
+              "HeIII_density",
+              "e_density",
+              "metal_density"];
+    }
+
+    derived {
+	field_list = ["temperature",
+		      "pressure",
+		      "cooling_time"];
+    }
+
+}
+
+Method {
+     list = [ "pm_deposit", "gravity", "ppm", "grackle", "pm_update", "comoving_expansion" ];
+    
+     output {
+        field_list += ["HI_density",
+	                 "HII_density",
+	                 "HeI_density",
+	                 "HeII_density",
+	                 "HeIII_density",
+	                 "e_density",
+                         "metal_density",
+	                 "cooling_time",
+	                 "temperature",
+	                 "gamma"]; 
+     } 
+     grackle {
+        courant = 0.50; # meaningless unless use_cooling_timestep = true;
+
+        data_file = "input/CloudyData_UVB=HM2012.h5";
+
+        with_radiative_cooling  = 1;
+        primordial_chemistry    = 1;  # 1, 2, or 3
+        metal_cooling           = 1;  # 0 or 1 (off/on)
+        UVbackground            = 1;  # on or off
+        metallicity_floor = 1e-6; # metallicity floor in solar units
+        cmb_temperature_floor   = 1;
+
+        self_shielding_method = 0;  # see pg. 23-24 of https://grackle.readthedocs.io/_/downloads/en/latest/pdf/
+
+        HydrogenFractionByMass = 0.73;
+
+        # set this to true to limit the maximum timestep to the product of the
+        # minimum cooling/heating time and courant.
+        use_cooling_timestep = false; # default is false
+     }
+}
+
+
+Output {
+     list = ["hdf5"];
+     hdf5 {
+          dir = ["Dir_COSMO_MULTI_%04d","cycle"];
+          field_list += ["HI_density",
+	                 "HII_density",
+	                 "HeI_density",
+	                 "HeII_density",
+	                 "HeIII_density",
+	                 "e_density",
+                         "metal_density",
+	                 "cooling_time",
+	                 "temperature",
+	                 "gamma"];
+        }
+     de   { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     depa { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     ax   { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     ay   { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     az   { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     dark { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     mesh { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     po   { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     hdf5 { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     dep  { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
+     check { dir = [ "Dir_COSMO_MULTI_%04d-checkpoint", "count" ]; }
```

Comment:
```suggestion
        }
     hdf5 { dir = [ "Dir_COSMO_MULTI_%04d", "cycle" ]; }
```

It might be helpful to remove these to avoid confusing new users (since they aren't used)

Reply:
Removed